Gruntfile.js

```diff
@@ -3,7 +3,7 @@ module.exports = function (grunt) {
 
     var bumpFiles = [
         'package.json',
-        'bower.json'
+        'package-lock.json'
```

Comment:
should this be a bump file or should you rerun npm install?

Reply:
I think that it should be bump. There is not necessarily a reason to run npm install when bumping and it would just be another thing to remember to do.